src/web/app/pages-admin/admin-accounts-page/admin-accounts-page.component.ts

```diff
@@ -56,8 +39,7 @@ export class AdminAccountsPageComponent implements OnInit {
    * Loads the account information based on the given ID.
    */
   loadAccountInfo(instructorid: string): void {
-    const paramMap: { [key: string]: string } = { instructorid };
-    this.httpRequestService.get('/accounts', paramMap).subscribe((resp: AccountInfo) => {
```

Comment:
We should not change this since we have agreed not to touch GET `/accounts`?

Reply:
I didn't change any logic of it, all I did was move it to `account-service.ts`, because I am in charge of all accounts related logic. Thus I feel it is more consistent this way.

Reply:
I don't think so. Previously, the violations just reside in this file. If we adopt the change, the violation span between account service and this file.

Reply:
Okay I will change it back, but intuitively It feels weird to make it half way done

Reply:
Remember not to convey changes that out of scope of the PR. Sometimes, you may find that you need to change everything. (and that's why the GET account API is classified as level 4 https://docs.google.com/spreadsheets/d/1RRHp-h29P406_SgmTpK8CvY7X0HvKOoXl-NYNsl9aD4/edit#gid=1084594222&range=D94)